x-pack/plugins/triggers_actions_ui/public/application/sections/alert_summary_widget/components/alert_summary_widget_loader.tsx

```diff
@@ -6,22 +6,26 @@
  */
 
 import React from 'react';
-import { EuiLoadingChart } from '@elastic/eui';
+import { EuiLoadingChart, EuiLoadingSpinner } from '@elastic/eui';
 import { AlertSummaryWidgetProps } from '..';
 
-type Props = Pick<AlertSummaryWidgetProps, 'fullSize'>;
+type Props = Pick<AlertSummaryWidgetProps, 'fullSize' | 'hideChart'>;
 
-export const AlertSummaryWidgetLoader = ({ fullSize }: Props) => {
+export const AlertSummaryWidgetLoader = ({ fullSize, hideChart }: Props) => {
   return (
     <div
       style={{
-        minHeight: fullSize ? 238 : 224,
+        minHeight: hideChart ? 44 : fullSize ? 238 : 224,
         display: 'flex',
         alignItems: 'center',
-        justifyContent: 'center',
+        justifyContent: hideChart ? 'flex-start' : 'center',
       }}
     >
-      <EuiLoadingChart size="l" data-test-subj="alertSummaryWidgetLoading" />
+      {hideChart ? (
```

Comment:
Maybe I'm missing some context but the prop name `hideChart` suggest that no element will be shown but instead it will show a spinner. Would it make to rename it to `isLoading` or `loadingChar` or something similar?

Reply:
`hideChart` prop will control if we want to show the chart under the alerts count or not so I use this prop here because when we have a chart we show a different spinner - so basically if there is no chart we want to show a small "round" spinner and not the big "chart" spinner.

Reply:
@jcger I renamed it to `isLoadingWithoutChart` to make it easier to understand that this is used for the loading when the cart is not there